modules/Other/components/Counter.jsx

```diff
@@ -2,24 +2,24 @@
 import React, { Component, PropTypes } from 'react';
 
 export default class Counter extends Component {
-  // const propTypes = {
-  //   increment: PropTypes.func.isRequired,
-  //   incrementIfOdd: PropTypes.func.isRequired,
-  //   decrement: PropTypes.func.isRequired,
-  //   counter: PropTypes.number.isRequired
-  // };
+  static propTypes = {
```

Comment:
I think we're refactoring to React.createClass() anyway so this won't matter so much.